src/components/picture/_children/source.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from "prop-types";
+import formatSrcset from "../../../utils/format-image-resizer-src";
+
+const Source = ({ height, media, resizedOptions, resizerURL, src, width, ...rest }) => {
+	if (!media || !src) return null;
+	if (resizedOptions?.auth) {
+		return (
+			<source
+				{...rest}
+				height={height}
+				media={media}
+				srcSet={formatSrcset(resizerURL.concat(src), resizedOptions, width, height)}
+				width={width}
+			/>
+		);
+	}
+	return <source {...rest} media={media} srcSet={src} height={height} width={width} />;
```

Comment:
The only thing that changes is what is the value of `srcSet` - what you think about the following? 

```suggestion
	return (
		<source
			{...rest}
			height={height}
			media={media}
			srcSet={
				resizedOptions?.auth
					? formatSrcset(resizerURL.concat(src), resizedOptions, width, height)
					: src
			}
			width={width}
		/>
	);
```

Reply:
This works for me... but I like fewer `return`s.

Reply:
Much cleaner! Thank you!